### PR TITLE
feat(js): added expo example app

### PR DESCRIPTION
**Pull request by blu3beri**
dependent on: #69 

Might need to do some work hot-reloading and everything but it works with a build.

Might be good if you review this as well @TimoGlastra

### CODE REVIEW

File: wrappers/javascript/react-native-app/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "react-native-app",
+  "version": "1.0.0",
+  "scripts": {
+    "start": "expo start --dev-client",
+    "android": "expo run:android",
+    "ios": "expo run:ios",
+    "build": "exit 0",
```
Comment (TimoGlastra) [inline wrappers/javascript/react-native-app/package.json:8]: I usually do "echo 'no build needed for react-native-app'" or something similar

File: wrappers/javascript/react-native/package.json [hunk 2]
```diff
@@ -18,13 +18,11 @@
   "files": [
```
Comment (TimoGlastra) [inline wrappers/javascript/react-native/package.json:18]: We should add the `native` directory here? Or is that part of another pr? As you alraedy updated the paths in the cmake / build.gradle files

**Review by TimoGlastra — APPROVED** [review]

File: wrappers/javascript/react-native/package.json (hunk 2 above)
Comment (blu3beri) [reply wrappers/javascript/react-native/package.json:18]: Did we not want to fetch them post install?

File: wrappers/javascript/react-native-app/package.json (hunk 1 above)
Comment (blu3beri) [reply wrappers/javascript/react-native-app/package.json:8]: Yeah I did `node -e console.log...` before (Windows does not have echo I believe), but that did not work. if we can do echo for windows I am okay with that.

Comment (TimoGlastra) [reply wrappers/javascript/react-native-app/package.json:8]: ah yeah not sure tbh. Let's stick with this

File: wrappers/javascript/react-native/package.json (hunk 2 above)
Comment (TimoGlastra) [reply wrappers/javascript/react-native/package.json:18]: You're totally right 🤦

File: wrappers/javascript/tsconfig.json [hunk 3]
```diff
@@ -2,11 +2,6 @@
   "extends": "./tsconfig.build.json",
   "compilerOptions": {
     "baseUrl": ".",
-    "paths": {
-      "aries-askar-shared": ["shared/src"],
-      "aries-askar-react-native": ["react-native/src"],
-      "aries-askar-nodejs": ["nodejs/src"]
-    },
```
[the remark this answers is not in the document]
Comment (blu3beri) [reply wrappers/javascript/tsconfig.json]: Yeah I was a bit confused with the tsconfigs and which did what. You are correct (just checked) that this is required for the editor.

**Comment by TimoGlastra** [comment]
Python CI is suddently failing?

**Review by TimoGlastra — APPROVED** [review]

**Comment by blu3beri** [comment]
Yeah @andrewwhitehead is this the flaky Python test failing or is there something else going on?

**Comment by andrewwhitehead** [comment]
There's a semi random test failure on Ubuntu just as it shuts everything down. It seems like it might be fixed in sqlx 0.6 but that has a different issue on Windows (not releasing the database file on close).

**Comment by blu3beri** [comment]
Closing this as the example app is not stable and can be added later.

**Pull request closed without merging**